tests/storage/utils.py

```
@@ -504,3 +511,23 @@ def create_windows_directory(windows_vm: VirtualMachineForTests, directory_path:
+        storage_class_name: Name of the storage class to get the minimum PVC size for
+        client: DynamicClient for API operations
+
+    Returns:
+        The minimum supported PVC size string (e.g., "1Gi") from the storage profile annotation
+        'cdi.kubevirt.io/minimumSupportedPvcSize', or None if not set
+    """
+    storage_profile = StorageProfile(name=storage_class_name, client=client, ensure_exists=True)
+    min_pvc_size = storage_profile.instance.metadata.get("annotations", {}).get(
+        f"{Resource.ApiGroup.CDI_KUBEVIRT_IO}/minimumSupportedPvcSize"
+    )
+    LOGGER.info(f"Minimum supported PVC size from the StorageProfile: {min_pvc_size}")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)